src/pages/resultsView/enrichments/EnrichmentsUtil.tsx

```diff
@@ -400,4 +406,109 @@ export function getGroupColumns(groups: { name: string, description: string }[],
         });
     });
     return columns;
+}
+
+export function getEnrichmentBarPlotData(data: AlterationEnrichmentRow[], genes: string[]): IMultipleCategoryBarPlotData[] {
+    const usedMajorCategories: {[gene:string]:boolean} = {};
```

Comment:
if it's category = gene, should we just called it usedGenes?

Reply:
+1 on domain specific naming. I always thought major/minor category language was pretty confusing but I couldnt think of a better way to describe it in the abstract setting